### ISSUE TITLE
Element clear not working

### ISSUE THREAD

**Issue by Semihal** [phalcon/cphalcon#12165]
Does not work in the form of a $elemnt->clear() while transfer in the form of any entity.

```php
        $confirmPassword = new Password('confirmPassword', [
            'placeholder'   => '***********',
        ]);

        $confirmPassword->addValidator(
            new Confirmation([
                'with'  => 'password',
                'message'   => 'Пароли должны совподать'
            ])
        );

        $confirmPassword->clear();
```
The field is not cleared. If only to refresh the page.

### Details
* Phalcon version: 3.0.0
* PHP Version: 7.0.8
* Operating System: Win10
* Installation type: installing dll
* Zephir version (if any): ---
* Server: Apache
* Other related info (Database, table schema): Mysql5.7

**Comment by sergeyklay** [comment]
@Semihal Could you please take a look at #12280? In general this is what you need. Right?

**Comment by sergeyklay** [comment]
Fixed in `3.0.x` branch.
Could you please check `3.0.x` branch?

```sh
git clone git@github.com:phalcon/cphalcon.git
cd cphalcon
git checkout 3.0.x

zephir fullclean
zephir build
```

**Comment by Semihal** [comment]
Yes, thank you
